examples/Makefile-nrf52840

```diff
@@ -43,7 +43,14 @@ GCCVersion                      = $(shell expr `$(CC) -dumpversion | cut -f1 -d.
 
 # Disable built-in mbedTLS by default allowing for nrf_security implementation with hardware acceleration.
 # When set to 0 built-in, software implementation would be used.
+ifeq ($(THREAD_VERSION), 1.2)
+ifeq ($(DISABLE_BUILTIN_MBEDTLS), 1)
+$(error for THREAD_VERSION>=1.2, cannot use CC310, please use DISABLE_BUILTIN_MBEDTLS=0)
+endif
+DISABLE_BUILTIN_MBEDTLS        = 0
+else
 DISABLE_BUILTIN_MBEDTLS        ?= 1
```

Comment:
Being unfamiliar with related aspects, I have some general questions:
- How are CC310 and builtin mbedtls different in performance? 
- Can we solve this issue by prohibiting ISRs when the main loop is using CC130

Reply:
> How are CC310 and builtin mbedtls different in performance?

In my test, the software approach is fast enough to encrypt the biggest packet(127 bytes) on time. However, according to @bukepo , the software approach is slow at calculating the eclipse curves. As only AES has the possibility of being called in interrupt context, I'm planning to only use builtin mbedtls for AES but leave all the other security calculation to CC310.

> Can we solve this issue by prohibiting ISRs when the main loop is using CC130

Unfortunately not. It is because the delayed handling of ACK will cause encryption problem to out going frame. The current sequence is like,
1. Transmit frame (or transmit ACK)
2. Frame header has been set, interrupt generated
3. Do encryption in this interrupt

Reply:
Thanks for explaination.